asreview/entry_points/simulate.py

```diff
@@ -148,7 +148,7 @@ def execute(self, argv):  # noqa
                 fp_tmp_simulation,
                 project_id=Path(args.state_file).stem,
                 project_mode="simulate",
-                project_name=Path(args.state_file).stem,
+                project_title=Path(args.state_file).stem,
```

Comment:
This is a breaking change in the documented API. We should make this backward compatible.